controllers/NewsController.js

```diff
@@ -0,0 +1,93 @@
+const db = require('../data/connect');
+
+async function getNews(req, res) {
+    try {
+        console.log('get News');
+        await db.query('SELECT * FROM news', (err, results) => {
+            if (err) {
+                throw e;
+            }
+            res.status(200).json(results.rows);
+        });
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+async function findNews(req, res){
+    try{
+        console.log('find News');
+        let title = req.query.title || "";
+        let author = req.query.author || "";
+        let description = req.query.description || "";
+     
+        await db.query(`SELECT * FROM news WHERE title LIKE '%${title}%' AND author LIKE '%${author}%' AND description LIKE '%${description}%'`, (err, results)=>{
```

Comment:
`pg` nó có hỗ trợ truyền tham số vào theo kiểu `$1`, `$2`... đấy, dùng kiểu đấy sẽ an toàn hơn là nối string.

Áp dụng cho cả các query khác `INSERT`, `UPDATE`...